applications/consdb/values-tucson-teststand.yaml

```diff
@@ -8,15 +8,24 @@ lfa:
 hinfo:
   latiss:
     enable: true
-    tag: "tickets-DM-44551"
+    tag: "24.10.1"
     logConfig: "consdb.hinfo=DEBUG"
   lsstcomcam:
     enable: true
-    tag: "tickets-DM-44551"
+    tag: "24.10.1"
     logConfig: "consdb.hinfo=DEBUG"
   lsstcam:
     enable: false
-    tag: "tickets-DM-44551"
+    tag: "24.10.1"
+
 pq:
   image:
-    tag: "main"
+    tag: "24.10.1"
+
+resources:
```

Comment:
Are these good choices for resource allocation?

Reply:
Certainly the memory seems overkill for pqserver. And the request memory seems excessive for both. What's the size of the containers?

Reply:
I would almost just remove these as we have not done a thorough review of the resources we need. I remember we added these when debugging because of seeing no space left on device while on TTS.